app/bundles/FormBundle/Entity/Field.php

```diff
@@ -1050,6 +1057,25 @@ public function isAutoFillReadOnly(): bool
         return $this->isAutoFill && $this->isReadOnly;
     }
 
+    /**
+     * Get field width.
+     */
+    public function getFieldWidth(): ?string
```

Comment:
```suggestion
    public function getFieldWidth(): ?string
```
No need for such comment as the method says it all :)